apps/ecparam.c

```diff
@@ -270,12 +280,6 @@ int ecparam_main(int argc, char **argv)
     if (out == NULL)
         goto end;
```

Comment:
Would it make more sense to also move this `bio_open_owner` call earlier too - rather that repeat the code?

Reply:
No, that would recreate the issue that #25552 was resolving.